website/docs/getting-started/installation.mdx

```diff
@@ -0,0 +1,135 @@
+
+You must explicitly allow the taq executable to run using the reccomended solutions detailed in the OS specific installation instructions
+:::
+
+Once these steps are complete, you will be able to run Taqueria from your terminal using the 'taq' command
+
+Step by step instructions for installing the Taqueria CLI on a Windows, MacOS, or Linux machine are detailed in the following section. If you are comfortable with downloading the unsigned Taqueria binary and configuring it to run in your terminal on your own, you can find the correct build for your local machine [here](https://github.com/ecadlabs/taqueria/releases), and then skip to the [Installing Taqueria Plugins Section](#installing-taqueria-pluginsz) 
+
+### OS Specific Installation Instructions
+
+To make installation as fast as possible, you can follow the steps detalled below for your particular operating system. Note that these steps are simply one method for installation. You can adjust the particular directories and shell configuration if you feel comfortable doing so
+
+The following steps assume the following terminals on each operating system:
+- MacOS: zsh
+- Windows: console???
+- Linux: bash
+
+
+<Tabs groupId="operating-systems">
+<TabItem value="mac" label="MacOS">
+  
+1. Create a directory to install the Taqueria binary and change into that directory
+
+2. Download the latest MacOS build of Taqueria from the [releases page](https://github.com/ecadlabs/taqueria/releases/) and name the binary file 'taq'
+```shell
+curl https://github.com/ecadlabs/taqueria/releases/download/untagged-d7ba60ff3f2af73f46c7/taqueria.v0.0.1-test.18-macos --output taq
+```
+
+3. Make the binary file executable
+```shell
+chmod -x taq
+```
+
+4. Add the installation directory to your $PATH
+
+5. Disable MacOS Gatekeeper to allow Taqueria to run
+```shell
+sudo spctl --master-disable
+```
+</TabItem>
+<TabItem value="win" label="Windows">Coming soon...</TabItem>
+<TabItem value="lnx" label="Linux">Coming soon...</TabItem>
+</Tabs>
+::: note
+    MacOS will give you a security warning and you need to approve the changes in order for Taqueria to run
+    :::
+
+## Installing Taqueria Plugins
+
+Taqueria plugins each implement a specific function such as a smart contract language, testnet, sandbox, or test framework. Taqueria plugins often rely on 3rd party software libraries such as LIGO or Docker which need to be installed on your machine for the plugin to work correctly
+
+Taqueria is designed to be flexible, streamlined, and customizable so plugins are installed on a per-project basis depending on the particular smart contract language, test framework, sandbox environment, and testnet used in the project
+
+Once installed, plugins provide their functionality by exposing tasks in the Taqueria CLI. The tasks available to run on a particular project can be viewed by running the ```shell taq --help ``` command in a project directory that's using Taqueria 
```

Comment:
triple backtick codefences only work with an argument on a new line 

    The quick ```shell brown``` fox <-- doesn't work as you expect

    ```shell
    $ echo "the quick brown fox"
    ```

In this body of text you should just use single back-ticks in the paragraph

    The quick brown `taq`uito